### ISSUE TITLE
Possible merge with pelotech/drone-helm3?

### ISSUE THREAD

**Issue by josmo** [bitsbeats/drone-helm3#1]
@foosinn Just wanted to start a conversation on if it made sense to merge this with the pelotech/drone-helm3 plugin since there's a decent amount of overlap :)

**Comment by foosinn** [comment]
hey,
limiting to only one helm3 plugin sounds good to me.

what where your reasons to create another version of it?

**Comment by josmo** [comment]
@foosinn cool!

 definitely didn't plan on having a second one. We started it before this one but then saw it when we went to publish in the registry. So just a timing thing.

**Comment by foosinn** [comment]
Code and config of both plugins differ quite a bit. We intentionally broke compatibility, since one needs to migrate anyway. Your approach differs :)

Right now i am not entirely sure if we want to give up control over the plugin. We had some struggles with outdated helm versions in the past.

**Comment by josmo** [comment]
Yeah we definitely took a different path (wanted to try to make it as painless for anyone currently using drone-helm and we had quite a few projects using it).  We've had probs with the outdated helm versions as well, which is why we setup the repo in a way that forking and enabling publishing of the forked plugin is trivial (using the secrets to define the dockerhub repo) until updates get merged in and doesn't hold folks back (although I think I'd added that to drone-helm a while back :)).

Either way totally get not wanting to give up control. I'm totally down finding a way (now or later) to not have issues with outdatedness.
